### DISCUSSION TITLE
MIT 6.824 Lab2 Raft 笔记 | XX’s Blog

### DISCUSSION THREAD

**Discussion by xxchan** [xxchan/xxchan.github.io#20] [Announcements]
https://xxchan.github.io/cs/2021/02/24/6.824-raft.html 

 Just some random thoughts of mine.

**Comment by izackwu** [comment]
手动点赞！

**Comment by fatFire** [comment]
golang 的 channel 不管有无缓冲都会阻塞。无缓冲channel会阻塞很好理解。有缓冲channel阻塞怎么理解？读的时候如果channel为空，阻塞；写的时候缓冲满，阻塞。是这样理解吗

**Reply by xxchan** [reply]
是的
